tests/celery_app/context_test.py

```diff
@@ -0,0 +1,48 @@
+import asyncio
+from unittest.mock import patch
+
+from superdesk.errors import SuperdeskError
+from superdesk.celery_app import HybridAppContextTask
+from superdesk.tests.asyncio import AsyncFlaskTestCase
+
+
+class TestHybridAppContextTask(AsyncFlaskTestCase):
+    async def test_sync_task(self):
+        @self.app.celery.task(base=HybridAppContextTask)
```

Comment:
Is `base=HybridAppContextTask` required here, when we're setting `celery.Task = HybridAppContextTask`?

Reply:
I added it for two reasons: 
- It is explicit therefore easier to understand what we're testing
- If for some reason the `celery.Task = ` value is different in the config as some point, the test of the `HybridAppContextTask` class will still work/pass.